addons/packages/pinniped/0.12.1/bundle/.imgpkg/images.yml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: imgpkg.carvel.dev/v1alpha1
+images:
+- annotations:
+    kbld.carvel.dev/id: docker.io/getpinniped/pinniped-server:v0.12.1
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: v0.12.1_vmware.1
+          url: projects-stg.registry.vmware.com/tkg/pinniped:v0.12.1_vmware.1
+  image: projects-stg.registry.vmware.com/tkg/pinniped@sha256:19cd1f5b18cad1772ec177b66cd8f8307197cb7dd276ee66d92561d3a856cabe
+- annotations:
+    kbld.carvel.dev/id: projects.registry.vmware.com/tce/dex:v2.30.2_vmware.1
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: v2.30.2_vmware.1
+          url: projects-stg.registry.vmware.com/tkg/dex:v2.30.2_vmware.1
+  image: projects-stg.registry.vmware.com/tkg/dex@sha256:2b0212eb1a6032c4512c00d8a5d9e98976ad8a0925938027ee01b925e5c4484a
+- annotations:
+    kbld.carvel.dev/id: projects.registry.vmware.com/tce/tanzu_core/addons/tkg-pinniped-post-deploy:v1.3.1
+    kbld.carvel.dev/origins: |
+      - resolved:
+          tag: v0.17.0-dev-31-gf9c9c28e
```

Comment:
This is the latest image that's been pushed via Harbor, but not sure if we want to use a dev version.

Reply:
Looks like a force push removed the images.yml file. The package needs that to be complete. I assume that file will be restored when the image version is sorted out?

Reply:
@seemiller correct, thanks for noting that!